uk.co.compucorp.civicrm.hrcore/hrcore.php

```diff
@@ -167,6 +167,19 @@ function hrcore_civicrm_pageRun($page) {
 }
 
 /**
+ * Implements hook_civicrm_navigationMenu().
+ *
+ * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_navigationMenu/
+ */
+function hrcore_civicrm_navigationMenu(&$params) {
+  $menuItemID = CRM_Core_DAO::getFieldValue('CRM_Core_DAO_Navigation', 'Contacts', 'id', 'name');
```

Comment:
same lines repeated twice for the two menus, this can be separated into new function called Rename menu label.

Also don't forget to change page titles too using word replacement, and while it would be fine for (Contact) to replace it with (Staff) .. I am not sure if this is fine for (Administer/Configure) since it is a more generic word in context of civicrm/civihr.